constants/chapter.json

```diff
@@ -135,12 +135,52 @@
     ]
   },
   {
+    "id": 5,
     "title": "Chapter 5",
     "subtitle": "Higher-order Functions",
     "image": "/chapter-5.jpg",
-    "exercises": "",
+    "exercises": "Flattening, Your own loop, Everything, Dominant writing direction",
     "link": "/chapter5",
-    "details": ""
+    "details": "Being able to pass function values to other functions is a deeply useful aspect of JavaScript. It allows us to write functions that model computations with “gaps” in them.",
+    "devOscar": [
+      {
+        "id": 1,
+        "title": "Flattering",
+        "details": "The header board is a representation of an array of arrays. The board below, concatenates all the elements of the header board into a single array and replace the emojis by a motivational phrase."
```

Comment:
A couple grammar mistakes in here. Pro tip: check any block of text that the user will see with [Grammarly ](https://www.grammarly.com/grammar-check)

Reply:
I do not know what is wrong but sometimes it fails, I will reinstall the extension